apps/gallery/style/gallery.css

```diff
@@ -894,13 +894,13 @@ html[dir="rtl"] .sliderlabel {
   background: url(images/slider-marker.png) no-repeat top right;
 }
 
-.sliderlabel:after {
+.sliderlabel bdi:after {
   content: attr(data-label);
   position: relative;
   text-align: center;
   top: 0.9rem;
   width: 1rem;
-  -moz-margin-start: calc(-50%  + 0.1rem);
+  margin: 0 calc(-50% + 0.1rem);
```

Comment:
Also, this would mean that you don't need to modify the margins, which would be good.